src/screens/Home/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import i18next from 'i18next';
+
+import logoWoloxImg from 'assets/LogoWolox.png';
+import LocalStorageService from 'services/LocalStorageService';
+
+import styles from './style.module.scss';
+
+function Home() {
+  const history = useHistory();
+  const onLogoutButtonClick = () => {
+    LocalStorageService.removeValue('token');
+    history.push('/');
+  };
+
+  return (
+    <nav className={`row space-around ${styles.navBar}`}>
+      <Link className={styles.logo} to="/home">
+        <img className={styles.logoImage} src={logoWoloxImg} alt="Wolox logo" />
```

Comment:
only when you have an app with multiple languages, it would be nice to have this with i18n. help with accessibility 
in the alt attribute

Reply:
Perfect!
I tried to use:
```JSX
alt={i18next.t('Home:logoAlt')}
```
But, I had an error related to the type

<img width="614" alt="type error" src="https://user-images.githubusercontent.com/17410581/149576569-eadc40a7-5531-4bc2-9eef-b6896a251054.png">

So, I decided to create a variable to specify the `string` type necessary to solve the error.
```Typescript
  const logoAlt: string = i18next.t('Home:logoAlt');
```
Let me know if it is ok to you.

Reply:
try to use it like this `alt={`${i18next.t('Home:logoAlt')}`}`